README.rst

```diff
@@ -28,6 +30,32 @@ Usage
     # or from stdin:
     cat clippings.txt | clippings -
 
+Programmatic Usage
+^^^^^^^^^^^^^^^^^^
+
+.. code:: py
+
+    from clippings import parse_clippings
+
+    my_clippings_fl = ...
+    parse_clippings(my_clippings_fl)
+
+Want to parse non-English clippings?
```

Comment:
It might not be worth the additional package size and testing requirements, as well as the obligation to include other languages in the same way - Spanish now, French later? But what about Japanese?! I think I'd be more inclined to allow users to do that dependency injection, what do you think?

Reply:
Yeah let's keep the dependency injection for now, and reconsider if there's more demand for language support in general